VirtualAssistantBusinessLogic/VirtualAssistantBusinessLogic/KnowledgeGraph/KnowledgeGraph.cs

```diff
@@ -0,0 +1,70 @@
+﻿using System.Linq;
+using VirtualAssistantBusinessLogic.SparQL;
+using System.Collections.Generic;
+
+namespace VirtualAssistantBusinessLogic.KnowledgeGraph
+{
+    /// <summary>
+    /// Class representing the knowledge graph, that uses a SparQL Connection to
+    /// find information from the knowledge graph
+    /// </summary>
+    public class KnowledgeGraph
+    {
+        public KnowledgeGraph(SparQLConnectionFactory sparQLConnectionFactory)
+        {
+            SparQLConnectionFactory = sparQLConnectionFactory;
+        }
+
+        private SparQLConnectionFactory SparQLConnectionFactory { get; set; }
+
+        /// <summary>
+        /// Takes a knowledge graph node and gets the relevant information for that node based on
+        /// the nodes type.
+        /// </summary>
+        /// <param name="node">a specific knowledge graph node [returned from a call to find nodes]</param>
```

Comment:
```diff
- /// <param name="node">a specific knowledge graph node [returned from a call to find nodes]</param>
+ /// <param name="node">a specific knowledge graph node [returned from a call to findNodes()]</param>
```

Reply:
Bør være call to FindNodes() hvis det skal stå som en method